HW04.md

```diff
@@ -0,0 +1,380 @@
+    ##                                                   race hispanic           place
+    ## 1                               Asian/Pacific Islander      100            Home
+    ## 2                                                White      100          Street
+    ## 3                                                White      100 Other specified
+    ## 4                                                White      100            Home
+    ## 5                                                White      100 Other specified
+    ## 6 Not Availabletive American/Not Availabletive Alaskan      100            Home
+    ##      education
+    ## 1          BA+
+    ## 2 Some college
+    ## 3          BA+
+    ## 4          BA+
+    ## 5       HS/GED
+    ## 6 Less than HS
+
+## Generating a data frame that summarizes the number of gun deaths per month and printing using `kable()`.
+
+Knitr is part of the knitr package. Allows you to change column names
+and add captions and make pretty tables in your knitted document. Hint:
+set the argument format = “markdown”
+
+``` r
+deaths_per_month <- gun_deaths %>%
+    count(month)%>%
+    rename(deaths = n)
+
+deaths_per_month<-transform(deaths_per_month, MonthAbb = month.abb[month])
+deaths_per_month$MonthAbb <- factor(deaths_per_month$MonthAbb, levels = c("Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"))
+
+deaths_per_month
+```
+
+    ##    month deaths MonthAbb
+    ## 1      1   8273      Jan
+    ## 2      2   7093      Feb
+    ## 3      3   8289      Mar
+    ## 4      4   8455      Apr
+    ## 5      5   8669      May
+    ## 6      6   8677      Jun
+    ## 7      7   8989      Jul
+    ## 8      8   8783      Aug
+    ## 9      9   8508      Sep
+    ## 10    10   8406      Oct
+    ## 11    11   8243      Nov
+    ## 12    12   8413      Dec
+
+### Generating a bar chart with human-readable labels on the x-axis. That is, each month should be labeled “Jan”, “Feb”, “Mar” (full or abbreviated month names are fine), not `1`, `2`, `3`.
+
+``` r
+ggplot(deaths_per_month, aes(MonthAbb, deaths))+
+    geom_bar(stat="identity")+
+    labs(x="Month", y="Deaths", title="Deaths Per Month")
+```
+
+![](HW04_files/figure-gfm/month_chart-1.png)<!-- -->
+
+``` r
+deaths_per_month
+```
+
+    ##    month deaths MonthAbb
+    ## 1      1   8273      Jan
+    ## 2      2   7093      Feb
+    ## 3      3   8289      Mar
+    ## 4      4   8455      Apr
+    ## 5      5   8669      May
+    ## 6      6   8677      Jun
+    ## 7      7   8989      Jul
+    ## 8      8   8783      Aug
+    ## 9      9   8508      Sep
+    ## 10    10   8406      Oct
+    ## 11    11   8243      Nov
+    ## 12    12   8413      Dec
+
+## Generating a bar chart that identifies the number of gun deaths associated with each type of intent cause of death. The bars should be sorted from highest to lowest values.
+
+``` r
+death_intent <- gun_deaths%>%
+    count(intent)%>%
+    rename(deaths=n)%>%
```

Comment:
It is a good idea to always rename these generated columns, which I failed to do almost every time. This certainly helps to explain what is going on here. Thanks.